src/ResponsiveProvider.js

```diff
@@ -115,7 +115,12 @@ const ResponsiveProvider = ({
     );
 
     useEffect(() => {
-        if (!process || !process.env || process.env.NODE_ENV !== 'test') {
+        if (
+            process &&
```

Comment:
Does webpack not catch this if optional chaining is used?

Also, we want it disabled in "test" so that it's not logged while running jest tests right?  Do you think an explanatory comment is needed here?

Reply:
If you use optional chaining that means that the client using this package needs that babel option.
And we would be requiring the developer to use what we want and it shouldn't be like that

Reply:
About the 'test' isn't it self-explanatory?

Reply:
I'm not understanding, the client using this package is only going to get what we build to.  Am I missing something?  Its not that important though.

With regards to the "test" part, I'm not talking about what it does, I have a suspicion about the motivation, and I'm asking for verification.  In addition, please see my below comment, you have a test saying `should call console.log only for development environment`.  Is there any reason the condition cannot be `if( process && process.env && process.env.NODE_ENV === 'dev')` ?

Reply:
Sorry you are right, we can add optional chaining in our babel config, got confused about the "webpack cache" what do you mean with it?

Regarding the test check my reply on your other comment

Reply:
Nice, cool 👍 

No, I said "catch" not "cache" 😄 .  

But never mind, I was confusing `process.env.NODE_ENV` with the variables from WebpackDefinePlugin.

Reply:
Well actually NODE_ENV gets set by define plugin with the value of the setting "mode" on webpack, it calls defineplugin under the hood